All EIPIP Meetings/Meeting 060.md

```diff
@@ -0,0 +1,121 @@
+
+**Greg:** specification section and the assets etc i want them to be comfortable that they can use that however they want like they don't need to worry the fact that cc0 doesn't give patent protection yes that's a concern and i mean we are not lawyers well maybe i am past life no i mean really you're entirely right greg like we only have like a very very surface level knowledge of any of this stuff so i mean so i spoke with lawyers in this and they also only have service level knowledge of this stuff i wouldn't give them too much credit um a big part of the problem is is most of these theoretical cases have never actually been tried in court and so everybody is guessing including the lawyers and the lawyers i guess you know with you know presumably history and background and knowledge of precedent and court cases and how courts have decided in the past but like there's no actual strong president that says most of this stuff the patent stuff i think actually has been tried in court and so
+
+**Micah:** [that](https://www.youtube.com/watch?v=OlKWquEmJms&t=6058)  is concerning i don't know if anyone's actually patent trolled cc0 before but people have patent trolled before like that is a thing that people have done i would just say i had thought we always did stick with cc0 so it's unfortunate if anything landed in assets otherwise and if it it can't be made cc0 then it should be linked to rather than copied assets um and then i don't i don't see the link out to something as anything like the same force as actual inclusion and but it would be good if that if the foundation wanted to hire a lawyer and if anything ever happened then you just point and say you know this this lawyer said it was okay we tried to do the right thing and that helps otherwise you know just pick a restrictive policy and be about it and quit worrying i'm afraid of being an so um if i have to maybe put it for the not writers do we have any consensus on this or we are still uh continuing discussion okay then probably i will add it to the next meeting and try to see what more for and against points do we have to consider yeah think there's one thing we can do let's split that uh issue or suggest to tim to split that issue into two so one is the licensing issue and that is the entire discussion separately is should we lent people's code and i think those two can be handled in isolation for each other and they may have very different resolutions is it something that is recommended to be done on uh ef devops channel or do you want it to be there on either side he has uh so right now he has on ethereum cat herders eipip he has an issue number 157 is linked from the agenda apparently that has two bullet points i think this should actually be split into two separate issues the discussions can remain separate 
```

Comment:
```suggestion
[that](https://www.youtube.com/watch?v=OlKWquEmJms&t=6058)  is concerning i don't know if anyone's actually patent trolled cc0 before but people have patent trolled before like that is a thing that people have done 

**Greg**: i would just say i had thought we always did stick with cc0 so it's unfortunate if anything landed in assets otherwise and if it it can't be made cc0 then it should be linked to rather than copied assets um and then i don't i don't see the link out to something as anything like the same force as actual inclusion and but it would be good if that if the foundation wanted to hire a lawyer and if anything ever happened then you just point and say you know this this lawyer said it was okay we tried to do the right thing and that helps otherwise you know just pick a restrictive policy and be about it and quit worrying

 **Micah**: I am not a fan of being an a** 
 
 **Pooja**: so um if i have to maybe put it for the not writers do we have any consensus on this or we are still continuing discussion
 
  **Micah**: No
  
 **Pooja**: okay then probably i will **add it to the next meeting** and try to see what more for and against points do we have to consider yeah think there's one thing we can do let's split that uh issue or **suggest to tim to split that issue into two so one is the licensing issue and that is the entire discussion separately is should we lint people's code and i think those two can be handled in isolation for each other and they may have very different resolutions**
 
**Pooja**:  is it something that is recommended to be done on uh ef devops channel or do you want it to be there on either side he has uh so 

**micah**: right now he has on ethereum cat herders eipip he has an issue number 157 is linked from the agenda apparently that has two bullet points i think this should actually be split into two separate issues the discussions can remain separate 
```